soperator/modules/active_checks/templates/upgrade_cuda.yaml.tftpl

```diff
@@ -33,17 +33,17 @@ resources:
                 for pkg in $pkgs; do
                   if apt-mark showhold | grep -q "^$pkg$"; then
                     echo "Unholding $pkg"
-                    sudo apt-mark unhold -o DPkg::Lock::Timeout=-1 "$pkg"
+                    sudo apt-mark unhold "$pkg"
                   fi
                 done
               done
 
               echo "Removing existing CUDA installation..."
-              sudo apt remove -o DPkg::Lock::Timeout=-1 -y cuda || true
+              sudo apt remove -y cuda || true
 
               echo "Installing CUDA version ${cuda_version}..."
-              sudo apt -o DPkg::Lock::Timeout=-1 update
-              sudo apt install -o DPkg::Lock::Timeout=-1 -y "cuda=${cuda_version}"
+              sudo apt update
+              sudo apt install -y "cuda=${cuda_version}"
```

Comment:
I'm not sure, that cuda package will install all this packages https://github.com/nebius/soperator/blob/e6c720f53046bfe340e19b482a24cddf3f5000eb/images/jail/jail.dockerfile#L58

Reply:
I unhold only these holds https://github.com/nebius/soperator/blob/e6c720f53046bfe340e19b482a24cddf3f5000eb/images/jail/jail.dockerfile#L72-L77